openmc/mgxs/mgxs.py

```diff
@@ -427,7 +430,11 @@ def get_mgxs(mgxs_type, domain=None, domain_type=None,
 
         Parameters
         ----------
-        mgxs_type : {'total', 'transport', 'nu-transport', 'absorption', 'capture', 'fission', 'nu-fission', 'kappa-fission', 'scatter', 'nu-scatter', 'scatter matrix', 'nu-scatter matrix', 'multiplicity matrix', 'nu-fission matrix', chi'}
+        mgxs_type : {'total', 'transport', 'nu-transport', 'absorption',
```

Comment:
This must actually be on one line in order for Sphinx to properly parse it. You're actually the third developer to try this, I wish we could add a comment stating that this must unfortunately remain on a single line for this reason, but this already is a comment!